sdcm/logcollector.py

```diff
@@ -697,7 +697,7 @@ def receive_log(node, remote_log_path, local_dir, timeout=300):
                                        timeout=timeout)
         return local_dir
 
-    def collect_logs(self, local_search_path: Optional[str] = None) -> Optional[str]:
+    def collect_logs(self, local_search_path: Optional[str] = None) -> Optional[list[str]]:
```

Comment:
You also need to change it on all other collect_logs of BaseLogEntity child classes

Reply:
BaseLogEntity has method collect, which return local dst where log file will be stored.